src/frontend/js/project-service.js

```diff
@@ -1082,4 +991,5 @@ angular.module('seashell-projects', ['seashell-websocket', 'marmoset-bindings',
       self.open = function(name, lock) {
         return (new SeashellProject(name)).init(lock);
       };
-    }]);
+    }
+  ]);
```

Comment:
Can you please describe what you changed in project-service.js?  It's really hard to review such a large diff.